src/controller/CHIPDeviceController.cpp

```diff
@@ -693,6 +694,7 @@ CHIP_ERROR DeviceCommissioner::PairDevice(NodeId remoteDeviceId, RendezvousParam
     CHIP_ERROR err                     = CHIP_NO_ERROR;
     Device * device                    = nullptr;
     Transport::PeerAddress peerAddress = Transport::PeerAddress::UDP(Inet::IPAddress::Any);
+    uint32_t mrpIdleInterval, mrpActiveInterval;
```

Comment:
nit: I guess those two variables always go with each other, so maybe it would be easier to create dedicated structure/class like `MRPIntervals` with idle and active members. I'm fine with leaving it as is as well.

Reply:
Yes, I completely agree and even such type exists: https://github.com/project-chip/connectedhomeip/blob/master/src/messaging/ReliableMessageProtocolConfig.h#L121, but this has the same source of a problem to the Damian's comment; https://github.com/kkasperczyk-no/connectedhomeip/pull/46#discussion_r743529250 In general GN forces to explicitly define dependencies between modules and currently we can't use elements of messaging module in the transport module. I will try to figure out something regarding this problem, but it's doesn't seem to be that easy.